### PR TITLE
Chore: add installation id summary

### PR DESCRIPTION
**Pull request by ifoukarakis**
#### Summary

Add latest summary for each installation ID.

### CODE REVIEW

File: transform/mattermost-analytics/models/marts/product/_product__models.yml [hunk 1]
```diff
@@ -355,4 +355,48 @@ models:
...
+      Summary of each installation based on the telemetry data. Contains data for servers with an existing 
+      installation id only.
+
+    columns:
+      - name: installation_id
+        description: The id of the installation.
+        tests:
+          - not_null
+          - unique
+      - name: installation_id
```
Comment (catalintomai) [inline transform/mattermost-analytics/models/marts/product/_product__models.yml:371]: duplicate?

Comment (ifoukarakis) [reply transform/mattermost-analytics/models/marts/product/_product__models.yml:371]: Hah, seems I accidentally added that using Github Copilot. Nice catch!

Comment (catalintomai) [reply transform/mattermost-analytics/models/marts/product/_product__models.yml:371]: Next step - adding comments (like this one) to PRs = )

**Comment by ifoukarakis** [comment]
Note that tests are failing due to installations ids existing in stripe but not in telemetry. We need to decide what to do with them.

**Review by nickmisasi — APPROVED** [review]
LGTM!

**Comment by ifoukarakis** [comment]
> Note that tests are failing due to installations ids existing in stripe but not in telemetry. We need to decide what to do with them.

Following discussion with @nickmisasi, there are reasons it would happen, but safe to ignore them. Removed check for now.

**Pull request merged**